uco-master/uco.ttl

```diff
@@ -12,14 +12,17 @@
 # imports: http://unifiedcyberontology.org/types
 # imports: http://unifiedcyberontology.org/victim
 
-@base <http://unifiedcyberontology.org/master> .
+@base <http://unifiedcyberontology.org/uco> .
+@prefix : <http://unifiedcyberontology.org/uco#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix victim: <http://unifiedcyberontology.org/master#> .
+@prefix uco: <http://unifiedcyberontology.org/master#> .
```

Comment:
I think this one should be "@prefix uco: <http://unifiedcyberontology.org/core#> ."

Reply:
the correct prefix for the uco.ttl file should be:
@prefix uco: <http://unifiedcyberontology.org/uco#>

Reply:
I'll run this down right now.

Reply:
@sbarnum   I fixed the error and committed... the commit message is messed up because I hit enter too soon. Sorry.

Reply:
looks good